src/Illuminate/Foundation/Http/Middleware/PreventRequestsDuringMaintenance.php

```diff
@@ -5,14 +5,16 @@
 use Closure;
 use Illuminate\Contracts\Foundation\Application;
 use Illuminate\Foundation\Http\MaintenanceModeBypassCookie;
+use Illuminate\Http\RedirectResponse;
+use Illuminate\Http\Request;
 use Symfony\Component\HttpKernel\Exception\HttpException;
 
 class PreventRequestsDuringMaintenance
 {
     /**
      * The application implementation.
      *
-     * @var \Illuminate\Contracts\Foundation\Application
+     * @var Application
```

Comment:
Can you undo all of these docblock changes?

Reply:
Beat me to it. Really need to take a look at my PhpStorm settings 😁 Reverted